sources/safe.move

```diff
@@ -594,6 +623,115 @@ public fun accept_ownership(safe: &mut BridgeSafe, ctx: &TxContext) {
     safe.roles_mut().owner_role_mut().accept_role(ctx)
 }
 
+// === Upgrade Management ===
+
+/// Initialize upgrade service with proper witness
+fun initialize_upgrade_service(safe: &mut BridgeSafe, ctx: &mut TxContext) {
+    safe.roles.owner_role().assert_sender_is_active_role(ctx);
+    let w = bridge_roles::grant_witness();
+    let (upgrade_service, _witness) = upgrade_service_bridge::new(
+        w,
+        ctx.sender(),
+        ctx,
+    );
+
+    // Store upgrade service in the safe's storage or pass it back to caller
```

Comment:
Here we say we store it or pass it back but we share it - we should update the comment